api/src/main/resources/workbench.yaml

```diff
@@ -1303,6 +1303,31 @@ paths:
           schema:
             $ref: '#/definitions/ReadOnlyNotebookResponse'
 
+  /v1/workspaces/{workspaceNamespace}/{workspaceId}/notebooks/{notebookName}/lockingMetadata:
+    parameters:
+      - $ref: '#/parameters/workspaceNamespace'
+      - $ref: '#/parameters/workspaceId'
+    get:
+      tags:
+        - workspaces
+      summary: Get the locking metadata for a notebook
+      description: |
+        Get the locking metadata for a notebook, if available.  If the notebook is not locked
+        (e.g. at first creation) this may be empty.  Additionally, the caller of this endpoint
+        may not have the appropriate permissions to determine who is locking this notebook.
+        When this is the case, the lastLockedBy field is set to UNKNOWN.
+      operationId: "notebookLockingMetadata"
```

Comment:
nit: most of our operation ids are verbs, since they translate into method names in swagger codegen. Of course, the endpoint immediately above this one violates this convention as well, so it could be classified as an existing issue.

Reply:
ok, updated